### PR TITLE
Minor changes on docs comments

### PR DESCRIPTION
**Pull request by curquiza**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -123,11 +121,10 @@ uploadArchives {
 	def ossrhUsername = findProperty('ossrhUsername')
 	def ossrhPassword = findProperty('ossrhPassword')
 
-	project.logger.lifecycle('Trying to upload files to Maven repository...')
-
 	repositories {
 		mavenDeployer {
 			beforeDeployment {
+				project.logger.lifecycle('Trying to upload files to Maven repository...')
 				MavenDeployment deployment -> signing.signPom(deployment)
```
Comment (eskombro) [inline build.gradle:128]:
```suggestion
			project.logger.lifecycle('Trying to upload files to Maven repository...')
			beforeDeployment {
				MavenDeployment deployment -> signing.signPom(deployment)
```

Comment (eskombro) [reply build.gradle:128]: This should fix the tests @curquiza

**Comment by curquiza** [comment]
You're the java saver 🦸
It's now ready for review!!

File: src/main/java/com/meilisearch/sdk/UpdateStatus.java [hunk 2]
```diff
@@ -37,23 +38,24 @@ public String toString() {
 
 	/**
 	 * Method to return the status of the Update Status object
-	 * @return String containing the status of the Update Status Object
+	 *
+	 * @return string containing the status of the Update Status Object
```
Comment (eskombro) [inline src/main/java/com/meilisearch/sdk/UpdateStatus.java:42]:
```suggestion
	 * @return String containing the status of the Update Status Object
```

File: src/main/java/com/meilisearch/sdk/UpdateStatus.java [hunk 3]
```diff
@@ -37,23 +38,24 @@ public String toString() {
...
 	 */
 	public String getStatus() {
 		return this.status;
 	}
 
 	/**
 	 * Method to return the id of the Update
-	 * @return String containing the identifier of the Update
+	 * @return string containing the identifier of the Update
```
Comment (eskombro) [inline src/main/java/com/meilisearch/sdk/UpdateStatus.java:50]:
```suggestion
	 * @return String containing the identifier of the Update
```

File: src/main/java/com/meilisearch/sdk/UpdateStatus.java [hunk 4]
```diff
@@ -69,39 +71,39 @@ public float getDuration() {
 
 	/**
 	 * Method to return the time that the Update was enqueued at
-	 * @return String value of the date and time of the Update when it was enqueued
+	 * @return string value of the date and time of the Update when it was enqueued
```
Comment (eskombro) [inline src/main/java/com/meilisearch/sdk/UpdateStatus.java:74]:
```suggestion
	 * @return String value of the date and time of the Update when it was enqueued
```

File: src/main/java/com/meilisearch/sdk/UpdateStatus.java [hunk 5]
```diff
@@ -69,39 +71,39 @@ public float getDuration() {
...
 	 */
 	public String getEnqueuedAt() {
 		return this.enqueuedAt;
 	}
 
 	/**
 	 * Method to return the time that the Update was processed at
-	 * @return String value of the date and time of the  Update when it was processed
+	 * @return string value of the date and time of the  Update when it was processed
```
Comment (eskombro) [inline src/main/java/com/meilisearch/sdk/UpdateStatus.java:82]:
```suggestion
	 * @return String value of the date and time of the  Update when it was processed
```

File: src/main/java/com/meilisearch/sdk/UpdateStatus.java [hunk 6]
```diff
@@ -69,39 +71,39 @@ public float getDuration() {
...
 	 */
 	public String getProcessedAt() {
 		return this.processedAt;
 	}
 
 	/**
 	 * Method to return the error code of the Update Status
-	 * @return String value of the error code of the Update Status
+	 * @return string value of the error code of the Update Status
```
Comment (eskombro) [inline src/main/java/com/meilisearch/sdk/UpdateStatus.java:90]:
```suggestion
	 * @return String value of the error code of the Update Status
```

File: src/main/java/com/meilisearch/sdk/UpdateStatus.java [hunk 7]
```diff
@@ -69,39 +71,39 @@ public float getDuration() {
...
 	 */
 	public String getErrorCode() {
 		return this.errorCode;
 	}
 
 	/**
 	 * Method to return the error type of the Update Status
-	 * @return String value of the error type of the Update Status
+	 * @return string value of the error type of the Update Status
```
Comment (eskombro) [inline src/main/java/com/meilisearch/sdk/UpdateStatus.java:98]:
```suggestion
	 * @return String value of the error type of the Update Status
```

File: src/main/java/com/meilisearch/sdk/UpdateStatus.java [hunk 8]
```diff
@@ -69,39 +71,39 @@ public float getDuration() {
...
 	 */
 	public String getErrorType() {
 		return this.errorType;
 	}
 
 	/**
 	 * Method to return the link of the error of the Update
-	 * @return String value of the errorLink of the Update
+	 * @return string value of the errorLink of the Update
```
Comment (eskombro) [inline src/main/java/com/meilisearch/sdk/UpdateStatus.java:106]:
```suggestion
	 * @return String value of the errorLink of the Update
```

File: src/main/java/com/meilisearch/sdk/Updates.java [hunk 9]
```diff
@@ -12,9 +12,10 @@ protected Updates(Config config) {
 
 	/**
 	 * Retrieves the Update at the specified iud with the specified updateId
+	 *
 	 * @param uid Index identifier to the requested Update
 	 * @param updateId Identifier of the requested Update
-	 * @return String containing the requested Update
+	 * @return string containing the requested Update
```
Comment (eskombro) [inline src/main/java/com/meilisearch/sdk/Updates.java:18]:
```suggestion
	 * @return String containing the requested Update
```

File: src/main/java/com/meilisearch/sdk/Updates.java [hunk 10]
```diff
@@ -24,8 +25,9 @@ String getUpdate(String uid, int updateId) throws Exception {
 
 	/**
 	 * Retrieves all Updates at the specified iud
+	 *
 	 * @param uid Index identifier to the requested Updates
-	 * @return String containing the requested Updates
+	 * @return string containing the requested Updates
```
Comment (eskombro) [inline src/main/java/com/meilisearch/sdk/Updates.java:30]:
```suggestion
	 * @return String containing the requested Updates
```

**Review by eskombro — APPROVED** [review]
Thanks @curquiza 

I would keep `String` with capital letter as it refers to a Java type, but this is just a suggestion, feel free to take it or not 👍🏽 

LGTM 🌮

File: src/main/java/com/meilisearch/sdk/Documents.java [hunk 11]
```diff
@@ -28,8 +29,9 @@ String getDocument(String uid, String identifier) throws Exception {
 
 	/**
 	 * Retrieves the documents at the specified uid
+	 *
 	 * @param uid Partial index identifier for the requested documents
-	 * @return A String containing the requested document
+	 * @return string containing the requested document
```
[the remark this answers is not in the document]
Comment (curquiza) [reply src/main/java/com/meilisearch/sdk/Documents.java:34]:
```suggestion
	 * @return string containing the requested document
```

**Review by eskombro — APPROVED** [review]
☕

**Comment by curquiza** [comment]
Bors merge

**Pull request merged**